metadata-schema/baseSchema.schema.json

```diff
@@ -24,65 +11,45 @@
         }
       ],
       "message": {
-        "format": "UUID must be a valid UUID"
+        "format": "ERROR_UUID_1"
       }
     },
     "file_path": {
       "type": "string",
       "minLength": 1,
       "message": {
-        "format": "Filepath must be provided"
+        "format": "ERROR_FILE_PATH_1"
       },
       "alternateKeys": [
         {
           "tdrFileHeader": "Filepath"
         }
-      ],
-      "fileExists": "This file does not exist in the list of records transferred."
+      ]
     },
     "date_last_modified": {
       "type": "string",
       "format": "date",
+      "message": {
+        "format": "ERROR_DATE_LAST_MODIFIED_1"
+      },
       "alternateKeys": [
         {
-          "tdrFileHeader": "ClientSideFileLastModifiedDate"
+          "tdrFileHeader": "Date Last Modified"
         }
       ],
-      "message": {
-        "format": "Date last modified must be a valid date"
-      }
-    },
-    "file_size": {
-      "type": [
-        "number",
-        "null"
-      ],
-      "tdrName": "ClientSideFileSize",
-      "tdrDescription": null
-    },
-    "date_created": {
-      "type": [
-        "string",
-        "null"
-      ],
-      "pattern": "^$|^(0?[1-9]|[12][0-9]|3[01])[\\/\\-](0?[1-9]|1[012])[\\/\\-]\\d{4}$|\\d{4}-[01]\\d-[0-3]\\dT[0-2]\\d:[0-5]\\d:[0-5]\\d(?:\\.\\d+)?Z?",
-      "tdrDescription": null,
-      "tdrName": "date_created"
-    },
-    "date_range": {
-      "type": "string"
     },
     "description": {
       "type": "string",
       "maxLength": 8000,
       "message": {
-        "format": "Description should be less than 8000 characters"
+        "format": "ERROR_DESCRIPTION_1"
       },
       "alternateKeys": [
         {
           "tdrFileHeader": "Description"
         }
-      ]
+      ],
+      "daDescription": "WARNING_DESCRIPTION_1"
```

Comment:
I don't understand how `daDesciption` relates to "warnings". Does it not make more sense to have this as:
...
"warning": "WARNING_DESCRIPTION_1"`